controllers/accountControllers/userAccountController.js

```diff
@@ -1,117 +1,110 @@
-const {Account} = require('../../models/userAccountModel');
-const {User} = require('../../models/userModel');
-const {validateAccount} = require('../../validation/validationAccount');
-
-
-const createAccount =  async (req, res, next) => {
-    const { error } = await validateAccount.validate(req.body);
-    if(error){
-        return next (new Error(error));
-    }
-    const {
-        accountName, 
-        accountCurrency, 
-        accountAmount, 
-        accountDescription, 
-        accountStat
-    }   = req.body;
-
-    const isAccount = await Account.findOne({accountName});
-
-    if(isAccount){
-        return next (new Error(`${isAccount.accountName} already exists.`));
-    }
-
-    const userID = req.user.id;
-    const account = await Account.create({
-        user: userID,
-        accountName, 
-        accountCurrency, 
-        accountAmount, 
-        accountDescription, 
-        accountStat,
-        accountCreated: new Date().toGMTString()
+const { Account } = require('../../models/userAccountModel');
```

Comment:
I recommend frontend and backend fixes deliver in separate PRs

Reply:
Sorry for the inconvenience. I'll make separate PRs next time